### ISSUE TITLE
Multiline arrays fail to parse

### ISSUE THREAD

**Issue by nwinter** [differentmatt/filbert#54]
```python
friends = [
    'Joan',
    'Ronan',
    'Nikita',
    'Augustus'
]
```

leads to a Filbert parser TypeError in [the demo page](https://rawgit.com/differentmatt/filbert/master/test/interactive.html) and dies on CodeCombat, too. Putting everything on one line works:

```python
friends = ['Joan', 'Ronan', 'Nikita', 'Augustus']
```

**Comment by differentmatt** [comment]
The fix for this is probably similar to #39

**Comment by Xavion3** [comment]
This is fixed by #75
